lib/config/configuration.go

```diff
@@ -1208,17 +1209,33 @@ func applyWindowsDesktopConfig(fc *FileConfig, cfg *service.Config) error {
 	}
 	ldapPassword, err := os.ReadFile(fc.WindowsDesktop.LDAP.PasswordFile)
 	if err != nil {
-		return trace.WrapWithMessage(err, "loading LDAP password from file %v",
+		return trace.WrapWithMessage(err, "error loading LDAP password from file %v",
 			fc.WindowsDesktop.LDAP.PasswordFile)
 	}
+
+	var cert *x509.Certificate
+	if !fc.WindowsDesktop.LDAP.SkipVerifyCA {
+		raw_cert, err := ioutil.ReadFile(fc.WindowsDesktop.LDAP.CAFile)
```

Comment:
```suggestion
		rawCert, err := os.ReadFile(fc.WindowsDesktop.LDAP.CAFile)
```

Generally avoid `_` in Go names, and prefer `os.ReadFile`.

Reply:
Brain fart, I've been writing python in my off-time.

Reply:
Why is `os.ReadFile` preferred?

Reply:
The `ioutil` package was [deprecated](https://golang.org/doc/go1.16#ioutil) in Go 1.16. All of its contents were moved into either pkg `os` or `io`.

`ioutil` will still exist for backwards compatibility, but all new code is encouraged to use the new locations.

Reply:
Gotcha, thank.